assisted_compliance.md

```diff
@@ -0,0 +1,45 @@
+## Title
+Helping repo owners be compliant by writing their CONTRIBUTING.md for them as a pull request
+
+### Alternate titles
+* Let us help you be compliant
+* Assisted compliance
+
+## Problem
+The team that owns the repository doesn't have a CONTRIBUTING.md; the task force needs them to have this to submit bug fixes.
+
+## Context
```

Comment:
The way the context is written make me think that the pattern only applies in a very narrow case. I feel we need to abstract a few things to make it more applicable.

Reply:
Can you suggest some ways to abstract it? Is the key restrictive context the mandatory nature of the CONTRIBUTING.md file?